lld/docs/ld.lld.1

```diff
@@ -719,6 +719,21 @@ a given pattern are handled as if they were given as arguments of
 Creates a separate output section for every orphan input section.
 .It Fl -unresolved-symbols Ns = Ns Ar value
 Determine how to handle unresolved symbols.
+.Ar value
+may be:
+.Pp
+.Bl -tag -width 2n -compact
+.It Cm report-all
+Report unresolved symbols (default).
+.It Cm ignore-in-object-files
+Only report unresolved symbols contained in shared libraries. Ignore if the are
```

Comment:
typo: they/they

Actually, suggest "Report unresolved symbols from shared libraries." and omit "Ignore ..."